code/modules/mob/mob.dm

```diff
@@ -1182,6 +1182,8 @@ GLOBAL_LIST_INIT(slot_equipment_priority, list( \
 			if(!no_text)
 				visible_message("<span class='warning'>[src] pukes all over [p_them()]self!</span>","<span class='warning'>You puke all over yourself!</span>")
 			location.add_vomit_floor(TRUE)
+	if (nutrition > 0)
+		adjust_nutrition(rand(-30, -15)) //❤ Monty unk
```

Comment:
don't add code comments like this.

Reply:
@hal9000PR please rereview removed the comment